src/ebmc/ebmc_base.cpp

```diff
@@ -346,14 +346,14 @@ bool ebmc_baset::parse_property(
   debug() << "Property: " << expr_as_string << eom;
   debug() << "Mode: " << transition_system.main_symbol->mode << eom;
 
-  properties.push_back(propertyt());
-  properties.back().expr=expr;
-  properties.back().expr_string=expr_as_string;
-  properties.back().mode = transition_system.main_symbol->mode;
-  properties.back().location.make_nil();
-  properties.back().description="command-line assertion";
-  properties.back().name="command-line assertion";
-  
+  properties.properties.push_back(propertyt());
+  properties.properties.back().expr = expr;
+  properties.properties.back().expr_string = expr_as_string;
+  properties.properties.back().mode = transition_system.main_symbol->mode;
+  properties.properties.back().location.make_nil();
+  properties.properties.back().description = "command-line assertion";
+  properties.properties.back().name = "command-line assertion";
```

Comment:
For a follow-up PR: `property` really wants a proper constructor to avoid this incremental build.